test/intl402/NumberFormat/prototype/format/useGrouping-de-DE.js

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 the V8 project authors. All rights reserved.
```

Comment:
Suggestion: since you're including multiple locales, I think it would be useful to include "en-IN", which triggers the crore-style grouping separators.  You probably don't need too many other locales for this feature.